schemas/context/profile-lead-crm.schema.json

```diff
@@ -0,0 +1,69 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-lead-crm",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Business Lead ",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/profile"
+  ],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/profile-work-details",
+    "https://ns.adobe.com/xdm/context/profile-other-work-details",
+    "https://ns.adobe.com/xdm/context/profile-personal-details",
+    "https://ns.adobe.com/xdm/context/profile-person-details",
+    "https://ns.adobe.com/xdm/context/profile-owning-entities"
```

Comment:
Missing "$id": "https://ns.adobe.com/xdm/common/contact-lead"

Reply:
We don't need to add a referred data type in the allOF section. That's not required.